examples/javascript-use-cjs/index.ts

```diff
@@ -14,9 +14,11 @@ const jsonSchemaDraft7 = {
           type: 'string',
           format: 'email'
         }
-      }
+      },
+      required: ['email']
```

Comment:
To stay consistent I don't think we should add anything unnecessary to the inputs, as you already have a test in the JavaScriptGenerator 🙂